Framework/CurveFitting/test/Algorithms/VesuvioCalculateMSTest.h

```diff
@@ -77,7 +79,7 @@ createTestWorkspace(const bool detShape = true,
       // Add another detector in the same position as the first
       auto shape = ShapeFactory().createShape(shapeXML);
       Mantid::Geometry::Detector *det2 = new Detector("det1", 2, shape, NULL);
-      det2->setPos(instrument->getDetector(1)->getPos());
+      det2->setPos(pos);
```

Comment:
We still need to do this for the test as we are creating a new detector and adding it to the instrument. I will add a comment to the test.